src/box/iproto_features.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2021, Tarantool AUTHORS, please see AUTHORS file.
+ */
+#include "iproto_features.h"
+
+#include <msgpuck.h>
```

Comment:
Could you please use `""` for non-system headers? Msgpuck here is not from the system, we always use the header stored in the source code.

Reply:
Okay.